sway/input/seatop_default.c

```diff
@@ -253,10 +253,47 @@ static void handle_tablet_tool_tip(struct sway_seat *seat,
  * Functions used by handle_button  /
  *--------------------------------*/
 
+static bool trigger_pointer_button_binding(struct sway_seat *seat,
+		struct wlr_input_device *device, uint32_t button,
+		enum wlr_button_state state, uint32_t modifiers,
+		bool on_titlebar, bool on_border, bool on_contents, bool on_workspace) {
+	// We can reach this for non-pointer devices if we're currently emulating
+	// pointer input for one. Emulated input should not trigger bindings.
+	if (device->type != WLR_INPUT_DEVICE_POINTER) {
+		return false;
+	}
+
+	struct seatop_default_event *e = seat->seatop_data;
+
+	char *device_identifier = device ? input_device_get_identifier(device)
+									 : strdup("*");
```

Comment:
Style: never align, instead use one or two tabs to indent continuation lines.

Reply:
Thanks for noticing, looks like I had "Reformat on paste" enabled in my editor by default :/